src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/RestoreAzureRMRecoveryServicesBackupItem.cs

```diff
@@ -133,6 +133,13 @@ public class RestoreAzureRmRecoveryServicesBackupItem : RSBackupVaultCmdletBase
         [ValidateNotNullOrEmpty]
         public string TargetFolder { get; set; }
 
+        /// <summary>
+        /// Array of source file paths to be recovered
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = AzureFileParameterSet,
+            HelpMessage = ParamHelpMsgs.RestoreFS.MultipleSourceFilePaths)]
+        public List<string> MultipleSourceFilePaths { get; set; }
```

Comment:
Please use single param name, aka `MultipleSourceFilePath`, see https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/cmdlet-best-practices.md#specific-noun-and-noun-singularity (sorry I forgot to mention this in the design review)

Reply:
Besides, please use `string[]` instead of `List<string>`. It's more friendly for the user.